toc/working-groups/foundational-infrastructure.md

```diff
@@ -103,8 +103,16 @@ areas:
   approvers:
   - name: Andrew Garner 
     github: abg
-  - name: Shaan Sapra
-    github: ssapra
+  - name: Colin Shield
+    github: colins
+  - name: Kyle Ong
+    github: ohkyle
+  - name: Kim Basset
```

Comment:
Based on the supplied contribution statistics Kevin and Ryan won't be eligible yet for approver status. They do however qualify for reviewer status. 

```suggestion
  - name: Kim Basset
  reviewers:
```